src/components/SummaryBox/SummaryBoxHeading/SummaryBoxHeading.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface SummaryBoxHeadingProps {
+  heading: string
+  className?: string
+  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
+}
+
+export const SummaryBoxHeading = ({
+  heading,
+  className,
+  headingLevel = 'h3',
```

Comment:
@suzubara @haworku @kylehilltruss What are your thoughts on having a default value here? In the past we've done it as a way to preserve backwards compatibility (non-breaking changes) where we've introduced the ability to customize the heading level. 

In this case, this is a breaking change anyways since the SummaryBox API is changing to be more compositional in nature. Should we just rip off the bandaid and make it a mandatory prop? IMO I think we should, since there's no real significance to `h3` other than it is what the USWDS example happens to use.

Reply:
@kimallen also makes a good point [here](https://github.com/trussworks/react-uswds/pull/1929#issuecomment-1081003074) about multi-staging this process.

Reply:
As long as we're not taking the USWDS example as the source of truth, then yes, I think we should remove the default value

Reply:
Yes to removing default value-  this is a breaking change we should put things in the ideal state we would like it to be in going forward.